README.md

```diff
@@ -27,6 +27,9 @@ config :bugsnag, notify_release_stages: ["production"]
 
 # Set `use_logger: true` to report all uncaught exceptions (using Erlang SASL)
 config :bugsnag, use_logger: true
+
+# Override the default bugsnag notify url
+config :bugsnag, notify_url: "https://notify.bugsnag.com"
```

Comment:
I'm thinking we should call this `endpoint_url`, since that's what Bugsnag's official docs call it, and it would be nice to align with those. Sorry for the additional churn 🙇

Reply:
no prob!